opentelemetry-appender-tracing/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 ### Added
 
 - Add log appender versions to loggers (#1182)
+- Normalized metadata fields with events from the log crate [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
+- Remove rendudant metadata [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
```

Comment:
```suggestion
- Remove redundant metadata [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
```

Reply:
Please callout that this removal of redundant data is only applicable for users doing logging via log crate.

Or put the entire changelog under a heading "Fixes the following for events emitted using log crate", to make it easy to follow changelog.

Reply:
also, I'd suggest to mention that the changes are under feature flag, so folks reading the changelog can quickly discover that fact without having to open up the PR itself

Reply:
I updated the CHANGELOG.MD. What do you think?